qlib/utils/index_data.py

```diff
@@ -517,9 +519,7 @@ def values(self):
 
 
 class SingleData(IndexData):
-    def __init__(
-        self, data: Union[int, float, np.number, list, dict, pd.Series] = [], index: Union[List, pd.Index, Index] = []
-    ):
+    def __init__(self, data=None, index=None):
```

Comment:
Why do we have to remove the type hinting instead of Decorating it with Optional?